spec/lib/tariff_jsonapi_parser_spec.rb

```diff
@@ -0,0 +1,257 @@
+require 'spec_helper'
+
+RSpec.describe TariffJsonapiParser do
```

Comment:
Parsing fixtures sounds good, I'll change to that

I'm keen to have scenarios we can control rather than recorded responses. I tend to find VCR style fixtures just bit rot over time and end up not reflecting changing APIs. At least with specific scenarios its easier to tweak if circumstances change.

Reply:
Oh gosh. Yeah, VCR is super painful IMO